src/loaders.py

```diff
@@ -53,22 +116,24 @@ def __getitem__(self, index):
         else:
             raise NotImplementedError("Possible loading options: 'memory' and 'disk'!")
 
-        Xi = load_func(self.X, index)
+        Xi = load_func(self.X, index, filetype='png', grayscale=False)
 
         if self.y is not None:
-            Mi = load_func(self.y, index)
+            Mi = self.load_target(self.y, index, load_func)
 
-            Xi, Mi = from_pil(Xi, Mi)
-            Xi, Mi = self.image_augment_with_target(Xi, Mi)
+            Xi, *Mi = from_pil(Xi, *Mi)
```

Comment:
@taraspiotr I think this is cool, it shows the distinction between image and some number of  target masks